sql/layers/layers.go

```diff
@@ -93,3 +93,51 @@ func GetByStatus(db sql.Executor, status Status) (rst types.LayerID, err error)
 	}
 	return rst, nil
 }
+
+func setHash(db sql.Executor, field string, lid types.LayerID, hash types.Hash32) error {
+	if _, err := db.Exec(fmt.Sprintf(`insert into layers (id, %[1]s) values (?1, ?2) 
+	on conflict(id) do update set %[1]s=?2;`, field),
+		func(stmt *sql.Statement) {
+			stmt.BindInt64(1, int64(lid.Uint32()))
+			stmt.BindBytes(2, hash[:])
+		}, nil); err != nil {
+		return fmt.Errorf("update %s to %s: %w", field, hash, err)
+	}
+	return nil
+}
+
+// SetHash updates hash for layer.
+func SetHash(db sql.Executor, lid types.LayerID, hash types.Hash32) error {
+	return setHash(db, "hash", lid, hash)
```

Comment:
please use named constants for `hash` and `aggregated_hash`

Reply:
done